CHANGELOG/v1.9.0-beta.1.md

```diff
@@ -0,0 +1,441 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.28.x -> v1.31.x
+- Workload Cluster: v1.26.x -> v1.31.x
+
+## Deprecation and Removals Warning
```

Comment:
Just finished a draft for highlights and deprecation and removals. If possible please replace this section with the highlights and deprecation sections from here: https://github.com/kubernetes-sigs/cluster-api/issues/11092#issuecomment-2402397477